### ISSUE TITLE
Design Panel - Color Input with Color Picker integration

### ISSUE THREAD

**Issue by swissspidy** [GoogleForCreators/web-stories-wp#378]
## Feature Description

As per https://github.com/google/web-stories-wp/pull/271#issuecomment-589332857 it is easier to split this out to unblock the PR.

Once #271 and #180 are merged, the color input needs to be made functional and use some `react-color` helper utilities to help deal with alpha values.

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._
